package.json

```diff
@@ -38,6 +38,8 @@
   },
   "dependencies": {
     "@alleyinteractive/block-editor-tools": "^0.10.2",
+    "@lukeed/uuid": "^2.0.1",
+    "@simbathesailor/use-what-changed": "^2.0.0",
```

Comment:
It looks like this is used for debugging - can we remove it before shipping an update to the plugin?